### DISCUSSION TITLE
Release 0.2.3 - For loops

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#93] [Announcements]
Bug fixes:

* Memory corruption in division of large numbers (#70)
* Parsing `'X⁻¹+(Y²+Z³)'` resulting in syntax error (#76)
* Parsing `-80/60` resulting in `4/3` (sign lost) (#78)
* Large numbers written to state file had separators (#83)
* Render names on stack as `'ABC'` instead of 'ABC' (#84)
* Evaluate names as `'ABC'` and not `ABC` (#86)
* Directory corruption following `purge` (#87)
* Local blocks now always execute (#90)
* Correct spacing of control structures on stack

Features:

* Keyboard binding to create lists (#89)
* Implementation of `for` loops (#92)

Other:

* Solidified the test suite further, added many tests
* Added reference to FOSDEM talk to [README.md](README.md)
* Added Freedom Covenant to the repository

Signed-off-by: Christophe de Dinechin <christophe@dinechin.org>

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/v0.2.3'>Release 0.2.3 - For loops</a>.</em>

**Comment by balta1980** [comment]
Already on my calc
